README.md

```diff
@@ -21,6 +38,15 @@ To start the Docker container that you just bult run:
 ```bash
 docker run -p 8443:8443 -it endpoint_querier
 ```
+# Additional Services
+The data collected by the endpoint querier can then be collected by Prometheus, which can be written to a Postgres database using the Prometheus Postgres storage adapter. This data can ultimately be viewed in Grafana. Below is information about how to start these additional services.
+
+## Starting All Services Using docker-compose
+All of the required services to run the Lantern back end are contained in the docker-compose file.
+```bash
+docker-compose up
```

Comment:
I'd add a few other pointers, particularly pointing out the different between `down` and `stop`:

link to docker-compose docs: https://docs.docker.com/compose/reference
`docker-compose up -d` runs everything in the background
`docker-compose stop` stops everything but keeps the containers, volumes, etc.
`docker-compose start` restarts containers after having called `docker-compose stop`
`docker-compose down` stops containers and removes them as well as networks (looks like you can remove volumes but it's not by default).

Reply:
needed to modify grafana instructions to use `pg_prometheus:5432`